src/services/personalities/community_centric.py

```
@@ -0,0 +1,50 @@
+"""
+CommunityCentric personality: Specialist in community-based approaches.
+"""
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)